packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -283,25 +284,54 @@ the generated files). Instead, it is ${JSON.stringify(pj.main)}`);
   };
 
   launchRendererDevServers = async (logger: Logger): Promise<void> => {
-    const config = await this.configGenerator.getRendererConfig(this.config.renderer.entryPoints);
-    if (config.length === 0) {
+    const configs = await this.configGenerator.getRendererConfig(this.config.renderer.entryPoints);
+    if (configs.length === 0) {
       return;
     }
 
-    for (const entryConfig of config) {
+    const preloadPlugins: string[] = [];
+    let preloadEntriesWithConfig = 0;
+    for (const entryConfig of configs) {
       if (!entryConfig.plugins) entryConfig.plugins = [];
       entryConfig.plugins.push(new ElectronForgeLoggingPlugin(logger.createTab(`Renderer Target Bundle (${entryConfig.target})`)));
 
+      const filename = entryConfig.output?.filename as string;
+      if (filename.endsWith('preload.js')) {
```

Comment:
Since we're using `?.` above, `filename` can be `undefined` and we can get an error here:
```suggestion
      if (filename?.endsWith('preload.js')) {
```